HW8/FunctionsLoop.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+
+void countingInteger(int input) {
+	int positiveNum = 0;
+	int negativeNum = 0;
+	int zero = 0;
+	int num = 0;
+	std::cout << "Please, enter your integers " << input << " times: ";
+	for (int i = 0; i < input; i++) {
+		std::cin >> num;
+		if (num == 0) {zero++;}
+		else if (num > 0) { positiveNum++; }
+		else { negativeNum++; }
+	}
+	std::cout << "Positive: " << positiveNum << ", Negative: " << negativeNum << ", Zeroes: " << zero;
+	std::cout << std::endl;
+}
+int factorialLoop(int count) {
+	int factorial = 1;
+
+	if (count < 0) {
+		std::cout << "ERROR!!! Integer can't be less than 0!";
+		return 1;
+	}
+
+	for (int i = 1; i <= count; i++) {
+		factorial *= i;
+	}
+	std::cout << "Factorial of " << count << " is " << factorial;
```

Comment:
It shouldn't 'talk'. Just get input and return the result;
It shouldn't even compile properly because there's no return ❓